src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -50,11 +50,14 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="GenerateWindowsPdbsForAssemblyBeingTested" BeforeTargets="GenerateTestExecutionScripts">
+  <!-- *********************************************************************************************** -->
     <!-- As of 10/2017 OpenCover does not support portable PDBs, but we want the builds to generate
          portable PDBs.  Thus we generate windows PDBs from portable PDBs here.   Can be removed
          when OpenCover directly supports Portable PDBs (probably by early 2018) -->
 
+  <Target Name="GenerateWindowsPdbsForAssemblyBeingTested" BeforeTargets="GenerateTestExecutionScripts"
+    Condition="$(CodeCoverageEnabled) == 'true' and '$(TargetOS)'=='Windows_NT'">
```

Comment:
Not that we have a style guide, but I've usually seen chopping done like this in MSBuild:

```xml
<Target Name="GenerateWindowsPdbsForAssemblyBeingTested"
        BeforeTargets="GenerateTestExecutionScripts"
        Condition="'$(CodeCoverageEnabled)' == 'true' and '$(TargetOS)'=='Windows_NT'">
```